Source/WebCore/html/PermissionsPolicy.cpp

```diff
@@ -449,4 +485,23 @@ bool PermissionsPolicy::isFeatureEnabled(Feature feature, const Document& docume
     return computeFeatureEnabled(feature, document, document.securityOrigin().data(), shouldReportViolation);
 }
 
+uint32_t PermissionsPolicy::inheritedPolicyValue() const
+{
+    uint32_t result = 0;
+    m_inheritedPolicy.forEachSetBit([&](size_t index) {
+        result |= 1 << index;
+    });
+
+    return result;
+}
+
+HashMap<uint8_t, std::optional<HashSet<SecurityOriginData>>> PermissionsPolicy::toContainerPolicyValue(const PolicyDirective& policyDirective)
+{
+    HashMap<uint8_t, std::optional<HashSet<SecurityOriginData>>> result;
+    for (auto& [feature, allowlist] : policyDirective)
+        result.set(index(feature), allowlist.value());
```

Comment:
We lose typing, I am not sure why.
Should probably be add as well, no need for set.